scripts/build.d/pybind11

```diff
@@ -12,7 +12,7 @@ if [ "${SRCSYNC}" == "tarball" ] ; then
   pkgfn=${pkgfull}.tar.gz
   pkgurl=https://github.com/pybind/pybind11/archive/refs/tags/v${pkgver}.tar.gz
 
-  download_md5 $pkgfn $pkgurl 695df9e99d92cdaa7fd0449fc5f30c26
```

Comment:
You updated the hash without the version.

Reply:
Thanks @yungyuc . The version has been updated to 2.12.0